docs/db/week-3/coursework.md

```diff
@@ -0,0 +1,5 @@
+## Personal statement clinic (PD - 1 hr)
+
+Prepare a personal statement for your Linkedin profile.
```

Comment:
```suggestion
Prepare a professional statement for your Linkedin profile.
```

Reply:
Can we be more specific about what is expected here? Share something you achieved or learned in Tech or PD classes. 
Should we ask them to research some good statements beforehand to have examples?